composer.dev.json

```diff
@@ -6,32 +6,36 @@
         "composer/installers": "^1.9",
         "cweagans/composer-patches": "^1.6.0",
         "drupal/core-composer-scaffold": "^10.0",
-        "drupal/core": "9.5.x",
-        "drush/drush": "^10",
+        "drupal/core": "10.1.4",
+        "drupal/core-recommended": "10.1.4",
+        "drush/drush": "^11",
         "vlucas/phpdotenv": "^4.1",
         "webflo/drupal-finder": "^1.0"
     },
     "require-dev": {
         "behat/behat": "^3.5",
         "behat/mink-selenium2-driver": "^1.4",
+        "chi-teck/drupal-code-generator": "^2.6.2",
         "squizlabs/php_codesniffer": "3.7.1",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
         "drupal/coder": "^8.3.10",
-        "drupal/console": "^1.0",
-        "drupal/drupal-extension": "^4.1",
+        "drupal/console": "dev-feature/Symfony-config-d10-compatibility",
```

Comment:
Is there a release now that supports this? I expect since D10 isn't as new now, there may well be

Reply:
@krakerag there is still a open PR for this - https://github.com/hechoendrupal/drupal-console/pull/4348

Reply:
I'm sad that they haven't merged it yet, I guess that will have to do. As long as we lock a working version in the lock file we should be fine.